src/Compilers/Test/Core/Compilation/CompilationExtensions.cs

```diff
@@ -34,11 +34,13 @@ namespace Microsoft.CodeAnalysis.Test.Utilities
     public static class CompilationExtensions
     {
         internal static bool EnableVerifyIOperation { get; } =
-#if ROSLYN_TEST_IOPERATION
-            true;
-#else
-            !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ROSLYN_TEST_IOPERATION"));
-#endif
+            // PROTOTYPE: Re-enable
```

Comment:
What's the nature of failures / work required to fix?

Reply:
There are 3 tests where an assert fails in `ControlFlowGraphBuilder()` in the IOperation CI leg. I'm not sure yet what is required to fix those cases, but I wanted to handle that in a specific PR that implements IOperation and CFG for collection literals.